src/App.js

```diff
@@ -10,12 +10,33 @@ import Education from './pages/Education/Education'
 import Resume from './pages/Resume/Resume';
 import NotFound from './pages/404/404';
 import Footer from './components/Footer/Footer';
+import { Helmet } from "react-helmet";
 
 
 function App() {
   return (
     <>
       <div className="App">
+        <Helmet>
+          <title>John Doe | Home</title>
+          <meta charset="UTF-8"/>
+          <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+          <meta name="keywords" content="free portfolio template,portfolio template, portfolio website template, open source, open sourcing, contributing to open source" />
+          <meta name="description" content="This project serves free and beautiful portfolio template for everyone. Anyone can showcase their work including moocs, education, projects, experience,etc. to the world." />
+          <meta name="robots" content="index,follow" />
+          <meta name="Googlebot" content="index,follow" />
+          <meta name="AdsBot-Google" content="index,follow" />
+          <meta property="og:site_name" content="https://portfolio.smaranjitghose.codes/"/>
```

Comment:
https://ahrefs.com/blog/seo-meta-tags/

Use this to improve the overall SEO using the meta tags. Feel free to modify the content

Reply:
Okay. Working on the changes.